src/pages/Home.vue

```diff
@@ -154,6 +151,16 @@ export default {
     getUserIpAddress() {
       return this.sessionData ? this.sessionData.userIp : "";
     },
+
+    /** Apart from ID, are any extra inputs being validated. */
+    isExtraInputValidationsRequired() {
+      return this.getAuthType.split(",").length > 1;
```

Comment:
If this thing is being calculated at Home, and you're sending the value of this computed variable to the child `Entry`, then why aren't you using this value inside `Entry`?
Inside `Entry`, I saw that you're recalculating this information using `this.authType`

Reply:
Correct, made the change.